ethereum/networks/src/main/java/tech/pegasys/teku/networks/EphemeryNetwork.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Consensys Software Inc., 2024
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.networks;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.spec.config.SpecConfig;
+import tech.pegasys.teku.spec.config.SpecConfigLoader;
+import tech.pegasys.teku.spec.config.builder.SpecConfigBuilder;
+import tech.pegasys.teku.spec.networks.Eth2Network;
+
+public class EphemeryNetwork {
+  private static final long GENESIS_CHAINID = 39438135;
+  private static final long GENESIS_TIMESTAMP = 1720119600;
+  private static final int PERIOD = 28;
+  private static final long PERIOD_IN_SECONDS = (PERIOD * 24 * 60 * 60);
+  public static final long PERIODS_SINCE_GENESIS =
+      ChronoUnit.DAYS.between(Instant.ofEpochSecond(GENESIS_TIMESTAMP), Instant.now()) / PERIOD;
+
+  public static void updateConfig(final SpecConfigBuilder builder) {
```

Comment:
One thing that may be handy is if we expand this a tiny bit, and pass in TimeProvider... that would allow us to write better tests because we could show it handling specific inputs and wind forward periods of time...

If we can adjust the time we can poke at the exact second of change etc which allows for some interesting testing of boundaries if thats helpful... Basically in tests you can use StubTimeProvider and set the time to whatever you like, and generally speaking use SystemTimeProvider in the rest of the system...

At its simplest, you'd potentially have an updateConfig that takes a timeProvider, and have a default interface like this that calls through with a SystemTimeProvider
so you'd have 
```
public static void updateConfig(final SpecConfigBuilder builder) {
  return updateConfig(builder, new SystemTimeProvider());
}
static void updateConfig(final SpecConfigBuilder builder, final TimeProvider timeProvider);
```
then potentially we'd make `periodsSinceGenesis` a package private method that can use a timeProvider to compute the result (its only used in your test so package private should be fine)

In this way we could even write tests just for that periodsSinceGenesis fn and playing with the StubTimeProvider...